tests/components/diagnostics/__init__.py

```diff
@@ -39,6 +43,34 @@ async def get_diagnostics_for_config_entry(
     return cast(JsonObjectType, data["data"])
 
 
+async def snapshot_get_diagnostics_for_config_entry(
+    hass: HomeAssistant,
+    hass_client: ClientSessionGenerator,
+    config_entry: ConfigEntry,
+    snapshot: SnapshotAssertion,
+) -> JsonObjectType:
+    """Snapshot get_diagnostics_for_config_entry."""
+    snapshot_exclude_functions: list[PropertyFilter] = [
+        props("created_at", "entry_id", "modified_at")  # we don't care about these
+    ]
```

Comment:
I don't agree with this change. These don't have to be in the context of the config entry.

Let's not add more "helpers".